atom/common/resources/mac/MainMenu.xib

```diff
@@ -65,6 +65,32 @@
 									<reference key="NSOnImage" ref="229763992"/>
 									<reference key="NSMixedImage" ref="909111550"/>
 								</object>
+								<object class="NSMenuItem" id="1025936716">
+									<reference key="NSMenu" ref="110575045"/>
+									<string key="NSTitle">Open Recent</string>
+									<string key="NSKeyEquiv"/>
+									<int key="NSKeyEquivModMask">1048576</int>
+									<int key="NSMnemonicLoc">2147483647</int>
+									<bool key="NSIsHidden">YES</bool>
+									<reference key="NSOnImage" ref="229763992"/>
+									<reference key="NSMixedImage" ref="909111550"/>
+									<string key="NSAction">submenuAction:</string>
+									<object class="NSMenu" key="NSSubmenu" id="1065607017">
+										<string key="NSTitle">Open Recent</string>
+										<array class="NSMutableArray" key="NSMenuItems">
+											<object class="NSMenuItem" id="759406840">
```

Comment:
I believe this `Clear Menu` item can be given a special role as `clearrecent` so people can choose to have it as a submenu item or just leave it out? If it's to be more flexible there needs to be some method similar to `menuFromModel` that allows populating menu items from a template based on the singleton menu for recent documents.

Reply:
Addressed in https://github.com/electron/electron/pull/11166/commits/ca24a18bdcdaa89f056f2dfeeacdef80780bbaa5.